### PR TITLE
Updated doc page to include NGINX route table

### PR DESCRIPTION
**Pull request by jacquayj**
For microservice ID and debugging.

### CODE REVIEW

File: content/resources/developer/microservice.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+| peregrine-service            | /api/v0/submission/graphql       | https://github.com/uc-cdis/peregrine                                                       |
+| peregrine-service            | /api/v0/submission/getschema     | https://github.com/uc-cdis/peregrine                                                       |
+| pidgin-service               | /coremetadata/                   | https://github.com/uc-cdis/pidgin                                                          |
+| portal-service               | /                                | https://github.com/uc-cdis/data-portal                                                     |
+| prometheus-server            | /prometheus/                     | https://github.com/uc-cdis/cloud-automation/tree/master/kube/services/monitoring           |
+| sheepdog-service             | /api/                            | https://github.com/uc-cdis/sheepdog                                                        |
+| shiny-nb2-service            | ~ ^/(shiny\|explore)/             | https://github.com/uc-cdis/cloud-automation/tree/master/kube/services/shiny                |
```
Comment (abgeorge7) [inline content/resources/developer/microservice.md]: I believe the shiny services are deprecated

File: content/resources/developer/microservice.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+| peregrine-service            | /peregrine/_status               | https://github.com/uc-cdis/peregrine                                                       |
+| peregrine-service            | /peregrine/_version              | https://github.com/uc-cdis/peregrine                                                       |
+| peregrine-service            | /api/search                      | https://github.com/uc-cdis/peregrine                                                       |
+| peregrine-service            | /api/v0/submission/graphql       | https://github.com/uc-cdis/peregrine                                                       |
+| peregrine-service            | /api/v0/submission/getschema     | https://github.com/uc-cdis/peregrine                                                       |
+| pidgin-service               | /coremetadata/                   | https://github.com/uc-cdis/pidgin                                                          |
+| portal-service               | /                                | https://github.com/uc-cdis/data-portal                                                     |
```
Comment (abgeorge7) [inline content/resources/developer/microservice.md]: This one is a bit confusing, as we don't have a "portal service"

File: content/resources/developer/microservice.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+|------------------------------|----------------------------------|--------------------------------------------------------------------------------------------|
+| ambassador-service           | /lw-workspace/proxy/             | https://github.com/uc-cdis/cloud-automation/tree/master/kube/services/ambassador           |
+| arborist-service             | /gen3-authz                      | https://github.com/uc-cdis/arborist                                                        |
+| arborist-service             | ~ /authz/?                       | https://github.com/uc-cdis/arborist                                                        |
+| arborist-service             | /authz/resources                 | https://github.com/uc-cdis/arborist                                                        |
+| arborist-service             | /gen3-authz-test                 | https://github.com/uc-cdis/arborist                                                        |
+| arranger-service             | /api/v0/flat-search/             | https://github.com/uc-cdis/arborist                                                        |
```
Comment (abgeorge7) [inline content/resources/developer/microservice.md]: This says "arranger-service" but the GitHub repo is "Arborist"
we're also encouraging users not to use Arranger so maybe we can get rid of it?

Comment (jacquayj) [reply content/resources/developer/microservice.md]: Removed

File: content/resources/developer/microservice.md (hunk 2 above)
Comment (jacquayj) [reply content/resources/developer/microservice.md]: Just pulled these rows from the Nginx config directory, but I see how that could be confusing. Removed portal-service row.

File: content/resources/developer/microservice.md (hunk 1 above)
Comment (jacquayj) [reply content/resources/developer/microservice.md]: Removed deprecated service references.

**Comment by vzpgb** [comment]
Thank you, John!

**Pull request merged**